src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -1183,7 +1185,15 @@ export function getAlertContent(alert) {
     case ALERT_TYPES.droHearingNoShow:
       return {};
```

Comment:
@cmgiven Just confirming: We still don't have content for `scheduled_dro_hearing` and `dro_hearing_no_show`, right?

Reply:
That's right. These statuses will not be used for this month's release.

Reply:
Perfect. We'll just leave them blank like this for now.